lua/tardis/libraries/sh_settings.lua

```diff
@@ -80,66 +49,236 @@ function TARDIS:GetSettingsData()
     return self.SettingsData
 end
 
-function TARDIS:SetSetting(id,value,networked,broadcast)
+--------------------------------------------------------------------------------
+-- Accessing
+
+function TARDIS:SetSetting(id, value)
+    local data = self.SettingsData[id]
+    if not data then error("Requested setting ", id, " does not exist") end
```

Comment:
Did you mean to use concat here?

```suggestion
    if not data then error("Requested setting ".. id  .." does not exist") end
```

Reply:
Yes, fixed